### PR TITLE
Remove the custom typing for stable.d.ts that was wrong.

### PR DESCRIPTION
**Pull request by usergenic**
I was wrong and I broke the build.  This fixes the build.

### CODE REVIEW

File: packages/analyzer/src/model/warning.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@
  */
 
 import * as chalk from 'chalk';
-import stable from 'stable';
+import * as stable from 'stable';
```
Comment (rictic) [inline packages/analyzer/src/model/warning.ts]: This file isn't a legal ES6 module, which is why I was using the `import stable = require('stable');` form before.

**Comment by usergenic** [comment]
I'm pretty sure I forgot how to do programming things.

File: packages/analyzer/src/model/warning.ts (hunk 1 above)
Comment (usergenic) [reply packages/analyzer/src/model/warning.ts]: Here's the actual fix: https://github.com/Polymer/tools/commit/f8daf698d306140448c1a63439b80a2b2d14f672

Comment (usergenic) [reply packages/analyzer/src/model/warning.ts]: I am not going to try and monkey with this again.  Sorry!

Comment (usergenic) [reply packages/analyzer/src/model/warning.ts]: (new ConeOfShame()).apply(usergenic);

**Pull request merged**